src/main/java/ch/uzh/ifi/hase/soprafs22/game/Game.java

```diff
@@ -20,9 +23,12 @@ public class Game {
     private final Map<String, PlayerDecorator> playerMap;
     private GameMap gameMap;
     private int turnNumber;
-    private String[] turnOrder;
+    private final String[] turnOrder;
     private boolean running;
 
+    private final GameLogger logger;
```

Comment:
Why do we need two ? But I haven't seen this pattern before, so I don't understand it ^^

Reply:
Indeed, removed one. Thanks for the catch!